pallets/staking/src/lib.rs

```diff
@@ -735,6 +737,57 @@ pub mod pallet {
             // signers see https://github.com/entropyxyz/entropy-core/issues/985
             Ok(Pays::No.into())
         }
+
+        /// An on-chain hook for TSS servers in the signing committee to report other TSS servers in
+        /// the committee for misbehaviour.
+        ///
+        /// Any "conequences" are handled by the configured Slashing pallet and not this pallet
+        /// itself.
+        #[pallet::call_index(7)]
+        #[pallet::weight(<T as Config>::WeightInfo::report_unstable_peer(MAX_SIGNERS as u32))]
+        pub fn report_unstable_peer(
+            origin: OriginFor<T>,
+            offender_tss_account: T::AccountId,
+        ) -> DispatchResultWithPostInfo {
+            let reporter_tss_account = ensure_signed(origin)?;
+
+            // For reporting purposes we need to know the validator account tied to the TSS account.
+            let reporter_validator_id = Self::threshold_to_stash(&reporter_tss_account)
+                .ok_or(Error::<T>::NoThresholdKey)?;
+            let offender_validator_id = Self::threshold_to_stash(&offender_tss_account)
+                .ok_or(Error::<T>::NoThresholdKey)?;
+
+            // Note: This operation is O(n), but with a small enough Signer group this should be
```

Comment:
I would like a formal justification for the capitilization of Signer.  
```suggestion
            // Note: This operation is O(n), but with a small enough signer group this should be
```

Reply:
The storage struct is called `Signers`, QED? lol 

Pls don't make me run CI again for this 😅